airbyte-integrations/connectors/source-zendesk-support/unit_tests/unit_test.py

```diff
@@ -221,6 +238,9 @@ def test_parse_response(requests_mock):
 
 
 class TestAllStreams:
+
+    patch = patch.object(UserSubscriptionStream, "get_subscription_plan", return_value="Enterprise")
```

Comment:
why do we need it in self? it's a bit confusing as for me

Reply:
Corrected.